### PR TITLE
Stop displaying progress when Maven --batch-mode is specified

### PR DESCRIPTION
**Pull request by fralalonde**
Fix for #1513

### CODE REVIEW

**Comment by fralalonde** [comment]
CLA: `I signed it!`

File: jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java [hunk 1]
```diff
@@ -130,20 +133,26 @@ private static EventHandlers makeEventHandlers(
...
+    EventHandlers eventHandlers =
+        new EventHandlers()
+            .add(
+                JibEventType.LOGGING,
+                logEvent -> consoleLogger.log(logEvent.getLevel(), logEvent.getMessage()))
+            .add(
+                JibEventType.TIMING,
+                new TimerEventHandler(message -> consoleLogger.log(LogEvent.Level.DEBUG, message)));
+
+    if (session.getRequest().isInteractiveMode()) {
```
Comment (chanseokoh) [inline jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java]: We should still add the dumb plain progress messages (just a few lines of message when not using the fancy progress bars).

```java
    ConsoleLoggerBuilder logEventHandlerBuilder =
        (isProgressFooterEnabled()
                ? ConsoleLoggerBuilder.rich(singleThreadedExecutor)
                : ConsoleLoggerBuilder.plain(singleThreadedExecutor).progress(log::info))
```

So, the check should be done inside `isProgressFooterEnabled()`. Will need passing `session` to the method.

File: jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java [hunk 2]
```diff
@@ -146,7 +149,11 @@ private static EventHandlers makeEventHandlers(
                             update.getProgress(), update.getUnfinishedAllocations()))));
   }
 
-  private static boolean isProgressFooterEnabled() {
+  private static boolean isProgressFooterEnabled(MavenSession session) {
```
Comment (chanseokoh) [inline jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java]: Now is a good time to add a test for this method. You'll need to make it package-private and add the `@VisibleForTesting`.

```java
@VisibleForTesting
tatic boolean isProgressFooterEnabled(...) {
```

Comment (chanseokoh) [reply jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java]: Taken care of in #1525.

**Pull request merged**